src/orion/plotting/backend_plotly.py

```diff
@@ -0,0 +1,103 @@
+import pandas as pd
+import plotly.graph_objects as go
+
+
+def regret(experiment, order_by, verbose_hover, **kwargs):
```

Comment:
No docstring?

Reply:
Fixed in 8f4b1c4563c9ec86748c4201460aaf3c9b25bb8c